.github/workflows/parallel-maven.yml

```diff
@@ -268,12 +268,17 @@ jobs:
         uses: aquasecurity/trivy-action@master
         env:
           TRIVY_DOWNLOAD_JAVA_DB_ONLY: true
+          TRIVY_JAVA_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-java-db,ghcr.io/aquasecurity/trivy-java-db:1
         with:
           scan-type: image
+          timeout: 10m0s
           cache-dir: .trivy
 
       - name: Trivy Vulnerability Scan
         uses: aquasecurity/trivy-action@master
+        env:
+          TRIVY_SKIP_DB_UPDATE: true
```

Comment:
Also add `TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db,ghcr.io/aquasecurity/trivy-db` here

Reply:
If we download the DB in an explicit step, this shouldn;t be needed.